### PR TITLE
Pipe siad's log to disk.

### LINKED ISSUE #25: Capture siad's logs
When Sia is running in a container we don't see its output. Let's redirect both `stdout` and `errout` to files within the data directory.

### PR DESCRIPTION
**Pull request by ro-tex**
Closes #25

### CODE REVIEW

File: scripts/run.sh [hunk 1]
```diff
@@ -15,4 +15,6 @@ END
 # PID 1. We need that in order to have `siad` receive OS signals (e.g. SIGTERM)
 # on container shutdown, so it can exit gracefully and no data corruption can
 # occur.
-exec $SIAD_CMD "$@"
+exec $SIAD_CMD "$@" \
+  1>"$SIA_DATA_DIR/siad_stdout.log" \
+  2>"$SIA_DATA_DIR/siad_errout.log"
```
Comment (ro-tex) [inline scripts/run.sh]: Hm, would it make more sense to append to those files or just keep the latest log?

Comment (ro-tex) [reply scripts/run.sh]: This kind of resolved itself with the request to keep outputting the logs to `stdout` and `stderr`. We can't keep them between executions, so anyone who wants that will have to do it themselves.

**Review by interstella-5555 — COMMENTED** [review]
Personally I don't see a benefit in copying output log to files if it is already available - see https://stackoverflow.com/a/41147654
It is just sia in docker container and I would keep it as simple as possible and as familiar as possible to people used to run their services with docker. 

On a side note, there is a way to run the container with logging to local file in docker docs https://docs.docker.com/config/containers/logging/local/

**Comment by ro-tex** [comment]
Good point, @kwypchlo. There's a lot about Docker I don't know, thanks for helping out!

I removed the scripted log output and added a section to the README. I'd appreciate any input about that as well.

**Review by interstella-5555 — APPROVED** [review]

**Pull request merged**